clusterloader2/testing/density/config.yaml

```diff
@@ -37,16 +53,14 @@ steps:
     - basename: saturation-rc
```

Comment:
I don't see a good reason for introducing "Uniform100qps" tuningset.
To simplify the config, let's remove this and use Uniform5qps here too.

Reply:
OK. Done.